legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -192,7 +192,21 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
         elif legal_name != response_json['business']['legalName']:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
-        elif founding_date != response_json['business']['foundingDate']:
-            msg.append({'error': 'Founding date does not match with founding date from Colin.',
-                        'path': business_founding_date_path})
+        else:
+            # Check if founding date is None
+            if founding_date is None:
+                msg.append({
+                    'error': 'Founding date is missing in the filing data.',
+                    'path': business_founding_date_path
+                })
+            else:
+                # Truncate both dates to "YYYY-MM-DDTHH:MM:SS" for comparison
+                founding_date_truncated = founding_date[:19]
+                colin_founding_date_truncated = response_json['business']['foundingDate'][:19]
+
```

Comment:
Colin is  '2009-07-23T07:00:00-00:00', the filing json is '2009-07-23T07:00:00.000+00:00'
truncated after 19th

Reply:
This is not a good approach. Can you try this one
```suggestion
datetime.fromisoformat(founding_date) !=  datetime.fromisoformat(response_json['business']['foundingDate'])
```

Reply:
Updated, thanks. Only concern is whether Colin can have different time zone? Other than -00:00, like +03:00. If not I think this approach is safe